selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/GitCompareTest.java

```diff
@@ -231,14 +231,7 @@ public void checkCompareWithRevision() {
     git.setFocusOnLeftGitCompareEditor();
     git.setCursorToLine(2, LEFT_COMPARE_ST);
     git.typeTextIntoGitCompareEditor("//change content from compare editor");
-
-    try {
-      git.waitExpTextIntoCompareLeftEditor("//change content from compare editor");
-    } catch (TimeoutException ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known permanent failure https://github.com/eclipse/che/issues/11791");
```

Comment:
We need to mark the test being _UNDER_REPAIR_ as well, isn't it?

Reply:
This test method has other permanent failure #11769, so we cannot delete the _UNDER_REPAIR_.
We need to wait the resolving of it.